Api/RegisterInterface.php

```diff
@@ -15,7 +15,7 @@
  *
  * @category    Goeasyship
  * @package     Goeasyship_Shipping
- * @copyright   Copyright (c) 2018 Easyship (https://www.easyship.com/)
+ * @copyright   Copyright (c) 2022 Easyship (https://www.easyship.com/)
```

Comment:
why not 2023?

Reply:
no idea. 😂 I think probably because we started the update of the plugin from 2022.